h/static/scripts/test/widget-controller-test.js

```diff
@@ -100,6 +101,10 @@ describe('WidgetController', function () {
       focus: sinon.stub(),
     };
 
+    fakeSettings = {
+      annotations: true
```

Comment:
`settings.annotations` is actually an optional string (the annotation ID), rather than a boolean. For the purpose of the tests this works the same way, but it is confusing to use a different data type in tests than what the actual code works with.

Conceptually:

```ts
interface Settings {
  annotations?: string;
}
```